src/Symfony/Component/ErrorHandler/Resources/TentativeTypes.php

```diff
@@ -0,0 +1,1548 @@
+<?php
+
+namespace Symfony\Component\ErrorHandler\Resources;
```

Comment:
Having a class in the `Resources` subnamespace is inconsistent with everything in Symfony. The `Resources` folder is never used as a PSR-4 namespace folder (also, our namespaces are singular).

Reply:
Does this matter? this class is internal, and it's a resource class to me.
I thought it would better here than alone in a subfolder, or worse at the root of the component.

Reply:
var-exporter + http-client use `\Internal\`, perhaps it's more consistent structure.

Reply:
oh this is a generated resource file xD 

i do agree perhaps it's a bit awkward to rely on autoloading, rather than requiring it say explicitly. But i dont think it matters (much) :+1:

Reply:
Now moved to `Internal`, thanks for the idea.